src/_gettsim/parameters/arbeitsl_geld_2.yaml

```
@@ -176,114 +176,240 @@ regelsatz:
     1: 364
     2: 328
     3: 291
-    4: 287
-    5: 251
-    6: 215
+    4:
+      min_alter: 14
+      max_alter: 17
+      bedarf: 287
```

```suggestion
      betrag: 287
```
Or "satz", but "bedarf" seems very misleading. Sorry for noticing so late in the game!